config/config.go

```diff
@@ -739,6 +741,7 @@ var defaultConf = Config{
 		GrpcKeepAliveTime:    10,
 		GrpcKeepAliveTimeout: 3,
 		GrpcCompressionType:  "none",
+		PDServerTimeout:      3,
```

Comment:
Now configurations are suggested to be global variables. 
If you insist to make it a configuration in the config file, please also add it into config.toml.example.

Reply:
I'd rather put this as a configuration filed since the value of the `PDServerTimeout` may be various in different TiDB instances in the same cluster.